pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -180,30 +181,15 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 	}
 
 	// Define an engineRunner pod which is secondary-resource #1
-	engineRunner, err := newRunnerPodForCR(instance, appUUID, appExperiments)
-	if err != nil {
-		return reconcile.Result{}, err
-	}
-	// Define the engine-monitor service which is secondary-resource #2
-	engineMonitor, err := newMonitorServiceForCR(instance)
+	engineRunner, err := newRunnerPodForCR(engine)
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+	// Define the engine-exporter service which is secondary-resource #2
```

Comment:
Delete this line

Reply:
I think that comment would be needed for somebody new!
please check and let me know, if still needed